server/services/approverCaseloadService.ts

```diff
@@ -107,77 +75,91 @@ export default class ApproverCaseloadService {
 
     return caseload
       .map(offender => {
-        const responsibleCom = coms.find(
-          com =>
-            com.username?.toLowerCase() ===
-            offender.licences
-              .find(l => offender.licences.length === 1 || l.status !== LicenceStatus.ACTIVE)
-              .comUsername?.toLowerCase()
-        )
-
-        if (responsibleCom) {
-          return {
-            ...offender,
-            probationPractitioner: {
-              staffCode: responsibleCom.staffCode,
-              name: `${responsibleCom.staff.forenames} ${responsibleCom.staff.surname}`.trim(),
-            },
-          }
-        }
-
-        if (!offender.deliusRecord.staff || offender.deliusRecord.staff.unallocated) {
-          return {
-            ...offender,
-          }
-        }
-
         return {
-          ...offender,
-          probationPractitioner: {
-            staffCode: offender.deliusRecord.staff.code,
-            name: `${offender.deliusRecord.staff.forenames} ${offender.deliusRecord.staff.surname}`.trim(),
-          },
-        }
-      })
-      .map((c: ManagedCaseForApproval) => {
-        const licence = _.head(c.licences)
-        const releaseDate = licence?.releaseDate || selectReleaseDate(c.nomisRecord)
-        const urgentApproval = licence.isDueToBeReleasedInTheNextTwoWorkingDays
-        let approvedDate
-        if (licence.approvedDate) {
-          approvedDate = format(parseCvlDateTime(licence.approvedDate, { withSeconds: true }), 'dd MMMM yyyy')
-        }
-        return {
-          licenceId: licence.id,
-          name: convertToTitleCase(`${c.nomisRecord.firstName} ${c.nomisRecord.lastName}`.trim()),
-          prisonerNumber: c.nomisRecord.prisonerNumber,
-          probationPractitioner: c.probationPractitioner,
-          submittedByFullName: licence.submittedByFullName,
-          releaseDate: releaseDate ? format(releaseDate, 'dd MMM yyyy') : 'not found',
-          sortDate: releaseDate,
-          urgentApproval,
-          approvedBy: licence.approvedBy,
-          approvedOn: approvedDate,
-          isDueForEarlyRelease: licence.isDueForEarlyRelease,
+          ...offender.licence,
+          probationPractitioner: this.findProbationPractioner(offender, coms),
         }
       })
       .sort((a, b) => (a.sortDate?.getTime() || 0) - (b.sortDate?.getTime() || 0))
   }
 
+  private toApprovalCase(
+    offender: { nomisRecord: CvlPrisoner; deliusRecord: OffenderDetail },
+    licences: LicenceSummaryApproverView[]
+  ) {
+    const { nomisRecord } = offender
+    const offenderLicences = licences.filter(l => l.nomisId === nomisRecord.prisonerNumber)
+    const licence =
+      offenderLicences.length === 1
+        ? offenderLicences[0]
+        : // licences to approve will always be SUBMITTED, recently approved will either be APPROVED or ACTIVE
+          offenderLicences.find(l => l.licenceStatus !== LicenceStatus.ACTIVE)
```

Comment:
I rewrote this a little bit to be more explicit with a ternary rather than weird size / find thing that was done in a couple of places. 
```ts
offender.licences
    .find(l => offender.licences.length === 1 || l.status !== LicenceStatus.ACTIVE)